wire-ios-data-model/Source/UseCases/UpdateMLSGroupVerificationStatusUseCase.swift

```diff
@@ -19,13 +19,13 @@
 import Foundation
 
 // sourcery: AutoMockable
-public protocol MLSConversationVerificationStatusProviderInterface {
+public protocol UpdateMLSGroupVerificationStatusUseCaseProtocol {
 
-    func updateStatus(_ groupID: MLSGroupID) async throws
+    func invoke(groupID: MLSGroupID) async throws
 
 }
```

Comment:
I recently added  `func updateAllStatuses() async throws` to this protocol, which makes it difficult to have this as a use case